chainlink-testing-framework/setup-run-tests-environment/action.yml

```diff
@@ -5,6 +5,10 @@ inputs:
     required: false
     description: The command to download the go modules
     default: make download
+  go_version:
```

Comment:
By default we want to use the go_mod_path since it automatically gets the latest 1.20.x version. Lets remove the default here so we don't cause conflicts. Especially since this is fix for a temporary issue.